packages/cosmic-swingset/lib/ag-solo/fake-chain.js

```diff
@@ -92,13 +92,13 @@ export async function connectToFakeChain(basedir, GCI, delay, inbound) {
   const maximumDelay = (delay || PRETEND_BLOCK_DELAY) * 1000;
 
   const withBlockQueue = makeWithQueue();
-  const simulateBlock = withBlockQueue(async function unqueuedSimulateBlock() {
-    const actualStart = Date.now();
-    // Gather up the new messages into the latest block.
-    thisBlock.push(...intoChain);
-    intoChain = [];
+  const unhandledSimulateBlock = withBlockQueue(
+    async function unqueuedSimulateBlock() {
```

Comment:
Style nit, I'd be inclined to define this function normally around line 93, outside of the call to `withBlockQueue()`, unless there's some compelling reason to define it inline.

Reply:
The reason was so that the wrapped function had a name, but to ensure that it was impossible to obtain an unwrapped reference to it.